bio-interpro-panel.js

```diff
@@ -222,23 +222,23 @@ class BioInterproPanel extends PolymerElement {
 
   __handleSelect(e) {
     let id = e.detail.item.id;
-    let url = `https://www.ebi.ac.uk/interpro/beta/api/entry/interpro/${id}`;
+    let url = `https://www.ebi.ac.uk/interpro/api/entry/interpro/${id}`;
```

Comment:
It would be better if these kinds of links were maintained solely in the bio-link component, rather than spread out in multiple components. It would make it easier to maintain.